Vagrantfile

```diff
@@ -1,33 +1,13 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
- # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "hashicorp/precise64"
+  config.vm.box      = 'ubuntu/trusty32'
```

Comment:
Good question. No reason. Just have been use it on other boxes.

I did a quick search but couldn't find perfomance data pointing to one or another. Found a couple discussions though:

* [Is a 64 or 32-bit VirtualBox guest better on a 64-bit host?](http://serverfault.com/questions/286643/is-a-64-or-32-bit-virtualbox-guest-better-on-a-64-bit-host)
* [32 or 64 guest](https://forums.virtualbox.org/viewtopic.php?f=1&t=60924)

What they *clame* is that:

1. 32 bits applications has a smaller memory footprint.
2. theres no actual perfomance difference between both if the VM memory < 4gb.
3. 64 bits can be a bit (not significant) faster on new cpus due to some special virtualization instructions.

However theres no data. I can update it to 64 bits if you think it's best.